### PR TITLE
[BULK] DocuTune - Fix build validation issues: docs-link-absolute (part 3)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: api-reference/beta/resources/intune-enrollment-depiosenrollmentprofile.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ Inherits from [depEnrollmentBaseProfile](../resources/intune-enrollment-depenrol
 |enableAuthenticationViaCompanyPortal|Boolean|Indicates to authenticate with Apple Setup Assistant instead of Company Portal. Inherited from [enrollmentProfile](../resources/intune-enrollment-enrollmentprofile.md)|
 |requireCompanyPortalOnSetupAssistantEnrolledDevices|Boolean|Indicates that Company Portal is required on setup assistant enrolled devices Inherited from [enrollmentProfile](../resources/intune-enrollment-enrollmentprofile.md)|
 |isDefault|Boolean|Indicates if this is the default profile Inherited from [depEnrollmentBaseProfile](../resources/intune-enrollment-depenrollmentbaseprofile.md)|
-|supervisedModeEnabled|Boolean|Supervised mode, True to enable, false otherwise. See https://docs.microsoft.com/intune/deploy-use/enroll-devices-in-microsoft-intune for additional information. Inherited from [depEnrollmentBaseProfile](../resources/intune-enrollment-depenrollmentbaseprofile.md)|
+|supervisedModeEnabled|Boolean|Supervised mode, True to enable, false otherwise. See [Deployment guidance: Enroll devices in Microsoft Intune](/mem/intune/fundamentals/deployment-guide-enrollment)
```
Comment (Lauragra) [inline api-reference/beta/resources/intune-enrollment-depiosenrollmentprofile.md:43]: Global: Files prefaces with "intune" are autogenerated and therefore need to be handled differently. Can you please create separate PRs for intune file fixes? So remove the two intune files from this PR and then we can merge the other fixes. Thanks.

Comment (alexbuckgit) [reply api-reference/beta/resources/intune-enrollment-depiosenrollmentprofile.md:43]: Will do, thanks.

**Comment by alexbuckgit** [comment]
Hi @Lauragra - I've removed the "intune-" files.

Please merge, thanks!

**Pull request merged**